src/lib/datepicker/calendar.ts

```diff
@@ -174,9 +202,15 @@ export class MatCalendar<D> implements AfterContentInit, AfterViewChecked, OnDes
   /** An input indicating the type of the header component, if set. */
   @Input() headerComponent: ComponentType<any>;
 
+  /** An input indicating the type of the footer component, if set. */
+  @Input() footerComponent: ComponentType<any>;
```

Comment:
We should have a test that checks whether this component is projected correctly.